config/peerpods/podvm/azure-podvm-image-handler.sh

```diff
@@ -410,7 +410,64 @@ function add_image_id_annotation_to_peer_pods_cm() {
         "LATEST_IMAGE_ID=${IMAGE_ID}" ||
         error_exit "Failed to add the image id as annotation to peer-pods-cm configmap"
 
-    echo "Image id added as annotation to peer-pods-cm configmap successfully"
+    echo "Image id annotation added as annotation to peer-pods-cm configmap successfully"
```

Comment:
This line doesn't seem to require change since it already mentions the annotation or am I missing something ?

Reply:
Yeah..